phoebe/backend/backends.py

```diff
@@ -1151,16 +1161,47 @@ def _run_single_time(self, b, i, time, infolist, **kwargs):
             logger.debug("rank:{}/{} PhoebeBackend._run_single_time: calling system.populate_observables at time={}".format(mpi.myrank, mpi.nprocs, time))
             system.populate_observables(time, populate_kinds, populate_datasets)
 
+        # save temporary positions (for interferometry)
+        system.xi = xi
+        system.yi = yi
+        system.zi = zi
+        system.distance = b.get_value('distance@system')
+
         logger.debug("rank:{}/{} PhoebeBackend._run_single_time: filling packets at time={}".format(mpi.myrank, mpi.nprocs, time))
         # now let's loop through and prepare a packet which will fill the synthetics
         packetlist = []
+        previous = None
         for k, info in enumerate(infolist):
             packet = dict()
 
-            # i, time, info['kind'], info['component'], info['dataset']
+            #i, time, info['kind'], info['component'], info['dataset']
             cind = starrefs.index(info['component']) if info['component'] in starrefs else None
-            # ts[i], xs[cind][i], ys[cind][i], zs[cind][i], vxs[cind][i], vys[cind][i], vzs[cind][i]
+            #ts[i], xs[cind][i], ys[cind][i], zs[cind][i], vxs[cind][i], vys[cind][i], vzs[cind][i]
```

Comment:
what is changed here, just removing the space?  We don't (yet) adhere to codestyle checks, but I think the space after the comment is standard.

Reply:
Corrected.